scrap.js

```diff
@@ -1,6 +1,8 @@
 const puppeteer = require('puppeteer');
 const cheerio = require('cheerio');
 const iPhone = puppeteer.devices['iPhone X'];
+const { Youtube } = require('scrape-youtube');
+const youtube = new Youtube();
```

Comment:
move this constructor to the function that's using it!

Reply:
done